File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Ask the user for the input height
    // If the input isn't 1 or 8 or between the two ask again
    int height = 0;
    do 
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);
    
    //For i in range 1 to height
    //  For m in range I to height
    //      print a space
    //  For j in range 0 to I
    //      print 1 hastag
    //print new line
    
    for (int i = 1; i <= height; i++)
    {
        for (int m = height - i; m > 0; m--)
        {
            printf(" ");
        }
        for (int j = 0; j < i; j ++)
        {
            printf("#");
        }
        printf("\n");
    }
    
}

File: pset6/marioMore/mario.py
from cs50 import get_int

# Prompts user for the value for variable height
# Re-prompts user until they give a value that is between 1 and 8
while (True):

    height = get_int("Height: ")
    if (height >= 1 and height <= 8):
        break
    
for i in range(1, height + 1):
    
    # Prints out height - i spaces and i hastags each line
    print(" " * (height - i), end="#" * i)
    
    # Prints out 2 spaces and i hastags each line
    print(" ", "#" * i)
    